Robot_Engine/Convert/Private Helpers/Load.cs

```diff
@@ -111,6 +111,11 @@ public static void RobotLoad(this GravityLoad load, RobotSimpleCase sCase, Robot
 
         public static void RobotLoad(this PointLoad load, RobotSimpleCase sCase, RobotGroupServer rGroupServer)
         {
+            if (load.Force.Length() == 0 & load.Moment.Length() == 0)
```

Comment:
Minor detail: make sure to always use `&&` instead of `&`. 

The former stops the check as soon as a condition is false. The later checks everything and THEN makes the boolean operation so not as efficient. The only case where you might see `&` is for bitwise operation, otherwise always use `&&`.

Reply:
The more you know! Fixed now, thanks @adecler